test_scripts/Defects/7_0/3164_1_OnInteriorVehicleData_empty_module_data.lua

```diff
@@ -0,0 +1,86 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/3164
+--
+-- Description: Check the successful merge of empty module data with existed data in cache
+--
+-- Precondition:
+-- 1. SDL Core and HMI are started
+-- 2. RC app is registered and activated
+-- 3. PTU with permission for RC is performed
+-- 4. App is subscribed to <moduleType>
+
+-- Steps:
+-- 1. HMI sends OnInteriorVehicleData notification with empty control data for <moduleType>
+-- SDL does:
+-- - a. merge received data with existed data in cache
+-- - b. send OnInteriorVehicleData notification with data received from HMI to mobile app
+-- 2. Mobile app requests GetInteriorVehicleData(<moduleType>)
+-- SDL does:
+-- a. respond to GetInteriorVehicleData with initial data from cache
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('user_modules/sequences/actions')
+local rc = require('user_modules/sequences/remote_control')
+
+-- [[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+config.application1.registerAppInterfaceParams.appHMIType = { "REMOTE_CONTROL" }
+
+-- [[ Local Variables ]]
+local appId = 1
+local isNotSubscribed = false
+local moduleData = {
+  emptyRadioData = { moduleType = "RADIO", radioControlData = {}},
+  emptyAudioData = { moduleType = "AUDIO", audioControlData = {}},
+  emptyHmiSettingsData = { moduleType = "HMI_SETTINGS", hmiSettingsControlData = {}},
+  emptyClimateData = { moduleType = "CLIMATE", climateControlData = {}},
+  emptySeatData = { moduleType = "SEAT", seatControlData = {}},
+}
+
+-- [[ Local Functions ]]
+local function getModuleId(pModuleType)
+  return rc.predefined.getModuleControlData(pModuleType, 1).moduleId
+end
+
+local function OnInteriorVehicleData(pModuleData)
+  local rpc = "OnInteriorVehicleData"
+  local mobSession = common.mobile.getSession(appId)
+  local hmi = common.hmi.getConnection()
+  hmi:SendNotification(rc.rpc.getHMIEventName(rpc), { moduleData = pModuleData })
+  mobSession:ExpectNotification(rc.rpc.getAppEventName(rpc), { moduleData = pModuleData })
+end
+
+local function GetInteriorVehicleData(pModuleData)
```

Comment:
@GetmanetsIrina 
```suggestion
local function getInteriorVehicleData(pModuleData)
```

This comment relates to other scripts of the PR too.